gap 4.8.9

alex-konovalov opened the pull request:
GAP 4.8.9 has been published today (https://mail.gap-system.org/pipermail/forum/2017/005628.html). This is straightforward update of the details of the new source archive and version number.

sjackman commented:
`gap` is up for migration to Homebrew/core. Consider submitting this formula to https://github.com/Homebrew/homebrew-core. This repository Homebrew/science is being retired at the end of this year. See https://github.com/Homebrew/homebrew-science/issues/6331

```
install events in the last 90 days for gap
================================================================================
1 | homebrew/science/gap                                         |  84 |  52.17%
2 | homebrew/science/gap --with-packages                         |  77 |  47.83%
```

alex-konovalov commented:
@sjackman thanks, done - I've submitted https://github.com/Homebrew/homebrew-core/pull/21928.

Also, what exactly is the meaning of the percentage in the table of install events in your comment?

alex-konovalov commented:
@sjackman What exactly would be "retired" - will it be available but not updated any more? I wonder if it may be still possible to merge this PR into Homebrew/science and then I will work on the formula for the coming GAP 4.9 release (with new build system) to go to Homebrew-core, addressing suggestions by @fxcoudert there? 

As you can see from the history of this formula, it was not created by myself - I am merely maintaining it by updating release details, so I will be able to return to this only in a couple of weeks...

sjackman commented:
> Also, what exactly is the meaning of the percentage in the table of install events in your comment?

Percentage of the total number of installs: 84 / (84 + 77) = 52.17%

sjackman commented:
For future PRs, the preferred git commit log message is
```
gap 4.8.9
```

alex-konovalov commented:
@sjackman great, thanks!

sjackman commented:
Merged. Thanks for your contribution to Homebrew/science, Alexander.

sjackman commented:
> What exactly would be "retired" - will it be available but not updated any more?

Yes, correct.
